drivers/NFSSR.py

```diff
@@ -94,8 +94,11 @@ def load(self, sr_uuid):
 
         # Handle optional dconf attributes
         self.transport = DEFAULT_TRANSPORT
+        useIPv6 = util.get_ip_address_family(self.remoteserver) == 6
```

Comment:
This isn't a pythonic variable name. should be `use_ipv6`. Acceptable patterns being  `^[_a-z][a-z0-9_]*$`

Reply:
Also adding these additional conditionals has pushed the cognitive complexity of the method over the allowed limits. Probably pushing this bit and the bit below into a `self.set_transport(...)` method would make it more readable.